sda-commons-starter-web/src/main/java/org/sdase/commons/spring/boot/web/auth/opa/OpaExcludesAuthorizationManager.java

```diff
@@ -31,16 +31,18 @@ public class OpaExcludesAuthorizationManager
   public OpaExcludesAuthorizationManager(
       @Value("${opa.exclude.patterns:}") String excludedPathPatterns,
       @Value(API_DOCS_URL) String apiDocsUrl) {
-    if (excludedPathPatterns.isEmpty()) {
-      this.excludedPathPatterns.add(Pattern.compile(apiDocsUrl + "\\.(json|yaml)"));
-    } else {
+    if (!excludedPathPatterns.isEmpty()) {
       Stream.of(excludedPathPatterns.split(","))
           .filter(Objects::nonNull)
           .filter(p -> !p.isBlank())
           .map(String::trim)
           .map(Pattern::compile)
           .forEach(this.excludedPathPatterns::add);
     }
+
+    // Always add the apiDocsUrls
+    this.excludedPathPatterns.add(Pattern.compile(apiDocsUrl));
+    this.excludedPathPatterns.add(Pattern.compile(apiDocsUrl + ".yaml"));
```

Comment:
Could you inject the value instead by using referencing `Constant.DEFAULT_API_DOCS_URL_YAML`? We would use the same approach like for `API_DOCS_URL`.